src/Service/Ecr/README.md

```diff
@@ -0,0 +1,20 @@
+# AsyncAws ECR Client
+
+![](https://github.com/async-aws/ecr/workflows/Tests/badge.svg?branch=master)
+![](https://github.com/async-aws/ecr/workflows/BC%20Check/badge.svg?branch=master)
+
+An API client for ECR.
+
+## Install
+
+```cli
+composer require async-aws/ecr
+```
+
+## Documentation
+
+See https://async-aws.com/clients/ecr.html for documentation.
```

Comment:
Dont forget to add `docs/clients/ecr.md`.

Reply:
doh, i copied src/Service/.template when was behind master.
might start again.